homeassistant/components/zyxelt50/manifest.json

```diff
@@ -0,0 +1,13 @@
+{
+  "domain": "zyxelt50",
```

Comment:
I don't think we should add this under that name.
It should be just `zyxel` and supported models should be reflected in the documentation.